tsconfig.frontend.json

```diff
@@ -7,7 +7,8 @@
     /* Code generation */
     "inlineSources": true,
     "strictPropertyInitialization": false,
-    "lib": ["dom", "es5"]
+    "lib": ["dom", "es5"],
+    //"types": ["cypress"]
```

Comment:
This is me trying to get cypress to work in ts, but did nothing. Left it here to comment the try, but should remove before merging